skopt/utils.py

```diff
@@ -292,3 +292,75 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+
+def dims_aslist(params_space):
+    """Convert dictionary of parameter names : skopt.space.Dimension
+    a list of skopt.space.Dimension, that can be used in Optimizer.
```

Comment:
I will think about a different way to formulate this. Right now I had to read it several times despite knowing what the function does :-/

""Convert a dictionary of parameters to a list.

The list of dimensions is created from the values of the
dictionary, sorted by the name of the dimensions used
as keys.

Counterpart to `parameters_asdict`.
"""

Maybe this?